evm/evmone-evm384/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM ewasm/bench-build-base:1
+
+LABEL maintainer="Ewasm Team"
+LABEL repo="https://github.com/ewasm/benchmarking"
+LABEL version="1"
+LABEL description="Ewasm benchmarking (evmone-evm384)"
+
+RUN pip3 install durationpy pandas
+
+# install evmone (need CXXFLAGS=-w or the build fails)
+WORKDIR /root
+RUN git clone --recursive --single-branch --branch evm384 https://github.com/axic/evmone.git evmone-evm384-v1
+RUN cd evmone-evm384-v1 && mkdir build && \
+  cd build && CXXFLAGS="-w" cmake .. -DEVMONE_TESTING=ON && cmake --build . -- -j
```

Please use this in both places:

```suggestion
  cd build && CXXFLAGS="-w" cmake .. -DEVMONE_TESTING=ON && make -j4
```